.eslintrc.js

```diff
@@ -1,13 +1,32 @@
-{
-  "root": true,
-  "env": {
-    "browser": true
+module.exports = {
+  globals: {
+    jest: "writeable",
+    describe: "readable",
+    it: "readable",
+    expect: "writeable",
+    afterEach: "writeable",
+    beforeEach: "writeable",
+    beforeAll: "readable",
+    afterAll: "readable"
   },
-  "parser": "@babel/eslint-parser",
-  "ignorePatterns": ["*test.js", "*.jest.js", "*.stories*.js"],
-  "extends": ["airbnb", "plugin:prettier/recommended", "plugin:react-hooks/recommended"],
-  "plugins": ["import", "react", "jsx-a11y", "json", "markdown", "react-hooks"],
-  "rules": {
+  env: {
+    browser: true,
+    es2021: true
+  },
+  extends: ["plugin:react/recommended", "airbnb", "plugin:react-hooks/recommended"],
+  parserOptions: {
+    ecmaFeatures: {
+      jsx: true
+    },
+    ecmaVersion: 13,
+    sourceType: "module"
+  },
+  plugins: ["import", "react", "jsx-a11y", "json", "markdown"],
```

Comment:
I think we still want the `react-hooks` plugin.

Reply:
@sahariko 
![image](https://user-images.githubusercontent.com/60314759/149146770-b7a63e5a-f198-4bae-96db-1880f94e2db4.png)

Reply:
Aaaah I see, since we're not extending anything, it's not needed.